cmd/open_test.go

```diff
@@ -0,0 +1 @@
+package cmd
```

Comment:
I'd like to add tests after this is reviewed, to check that functionality is how we want it